stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -42,8 +42,7 @@ public class _AnyTensorHandle {
 @_fixed_layout // required because the compiler accesses cTensorHandle directly.
 public final class TensorHandle<Scalar> : _AnyTensorHandle
   where Scalar : AccelerableByTensorFlow {
-  @usableFromInline
-  init(owning cTensorHandle: CTensorHandle) {
+  public init(owning cTensorHandle: CTensorHandle) {
```

Comment:
I'd suggest prepending an underscore to the first argument label because this should never be accessed by the user or show up in code completion. Also feel free to file a bug to do this after this patch lands.

Reply:
I prepended an underscore.